### PR TITLE
Removed outcome defaults - Please Review

### PR DESCRIPTION
**Pull request by mphprogrammer**
1. Fixes issue with NULL values reported as 0 in outcome rates.
2. Fixes issue with Production GIDS and new GIDs - Some Accreditation caution flags overwritten by Sec702 caution flag logic (rather than ORing to existing caution flags...)

### CODE REVIEW

**Comment by mphprogrammer** [comment]
@saintsoup52 @saneshark Did find one issue with caution flag reason Accreditation (Probation/Show Cause) and caution flag not being set. This exists in both Old and New GIDS

**Comment by mphprogrammer** [comment]
@saintsoup52 @saneshark Disregard last remark - both old and new gids agree completely regarding caution flags and caution flag reasons.  Ordering of caution_flag_reasons and spaces after commas, and text case caused whatever differences were found - can find nothing else wrong at the moment...

**Comment by saneshark** [comment]
@mphprogrammer This LGTM please feel free to merge when you are ready.

**Review by saneshark — APPROVED** [review]

**Review by saneshark — CHANGES_REQUESTED** [review]
Dont change existing migration

**Comment by saneshark** [comment]
@mphprogrammer I spoke too fast. Please don't modify existing migrations. Create a new migration to change defaults instead.

File: db/schema.rb [hunk 1]
```diff
@@ -853,17 +853,17 @@
...
-    t.float    "persistance_rate_veteran_otb", default: 0.0
-    t.float    "graduation_rate_veteran",      default: 0.0
-    t.float    "transfer_out_rate_veteran",    default: 0.0
+    t.string   "facility_code",                null: false
+    t.float    "retention_rate_veteran_ba"
+    t.float    "retention_rate_veteran_otb"
+    t.float    "persistance_rate_veteran_ba"
+    t.float    "persistance_rate_veteran_otb"
+    t.float    "graduation_rate_veteran"
+    t.float    "transfer_out_rate_veteran"
```
Comment (saneshark) [inline db/schema.rb]: @mphprogrammer missing one here

**Review by saneshark — APPROVED** [review]

**Comment by saneshark** [comment]
LGTM

**Comment by saintsoup52** [comment]
Not sure where we are with this... but the latest "verify file" (see link below) is still very different then the one in the old DS.

**Comment by mphprogrammer** [comment]
@saintsoup The old gids data used in verify I produced myself from my local copy of the current Data service. Along the errors you had mentioned, verify.xslx didn't find any differences between institution_new.csv and institution_old.csv, what differences in these csvs did you find?

**Comment by mphprogrammer** [comment]
@saintsoup52 also, there was no "link below" :-)

**Comment by saintsoup52** [comment]
I just used the data dashboard "data validation" worksheet to compare the final products and those screenshots I sent you were the results. 

I created the data validation worksheet (which can be downloaded with the dashboards) to run a side by side comparison and detect where there are differences. It was to make sure there were no major variances when we update the data week to week.

**Comment by mphprogrammer** [comment]
Did you use the `institution_old.csv` and `institution_new.csv` files to do that comparison?

**Pull request merged**